src/main/resources/network/brightspots/rcv/config_file_documentation.txt

```diff
@@ -113,10 +113,59 @@ Config file must be valid JSON format. Examples can be found in the test_data fo
     set of configuration parameters that specify the tabulation rules
     the "rules" section contains the following parameters:
 
+      "randomSeed" required when tiebreakMode is random, previousRoundCountsThenRandom, or generatePermutation
+        the integer seed for the application's pseudorandom number generator
+        value: [0..2147483647]
+        if not supplied: null
+
       "tiebreakMode" required
-        how the program should decide which candidate to eliminate when multiple candidates are tied for last place among continuing candidates in a given round
+        how the program should decide which candidate to eliminate when multiple candidates are tied for last place
+        or which candidate to elect first when:
+          1) electionWinnerMode is set to multiSeatAllowOnlyOneWinnerPerRound, and
```

Comment:
Would this also apply if it is the final round even if multiSeatAllowOnlyOneWinnerPerRound is not the current mode?

Reply:
When multiSeatAllowOnlyOneWinnerPerRound isn't enabled and multiple candidates exceed the threshold in the same round, we consider them to be elected simultaneously. Their surplus redistributions happen after they've been all been elected.